src/pages/Admin/Venue/Details/VenueDetails.tsx

```diff
@@ -96,23 +99,51 @@ const VenueDetails: React.FC<VenueDetailsProps> = ({ venue }) => {
     getOwnersData();
   }, [owners, ownersData]);
 
-  if (!user) return null;
+  const openPreviewLandingPage = useCallback(() => {
+    window.open(`/v/${venue.id}`);
```

Comment:
I would move this to a separate service\util function\class to abstract from direct `window` usage. But it is possible that this can be done in a separate PR not to block this PR's features